src/gui/widgets/multi_page.cpp

```diff
@@ -267,20 +283,25 @@ namespace implementation
 {
 
 builder_multi_page::builder_multi_page(const config& cfg)
-	: implementation::builder_styled_widget(cfg), builder(nullptr), data()
+	: implementation::builder_styled_widget(cfg), builders(), data()
 {
-	const config& page = cfg.child("page_definition");
+	const config& page = cfg.child();
 
-	VALIDATE(page, _("No page defined."));
-	builder = std::make_shared<builder_grid>(page);
-	assert(builder);
+	for (const config& page : cfg.child_range("page_definition"))
+	{
+		auto builder = std::make_shared<builder_grid>(page);
+		assert(builder);
+		builders[page["id"]] = builder;
+	}
+	VALIDATE(!builders.empty(), _("No page defined."));
 
 	/** @todo This part is untested. */
 	const config& d = cfg.child("page_data");
 	if(!d) {
 		return;
 	}
 
+	auto builder = builders.begin()->second;
```

Comment:
the problem here is that it's unclear to me how it was supposed to work...  in paticular builder_multi_page ctors does some stuff with  counting colums and i don't know whether that's intentional or just badly copied form list_definition.